Sources/Fuzzilli/Core/HybridEngine.swift

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import Foundation
+
+public class HybridEngine: ComponentBase, FuzzEngine {
+    // The number of mutations to perform to a single sample per round
+    private let numConsecutiveMutations: Int
+
+    public init(numConsecutiveMutations: Int) {
+        self.numConsecutiveMutations = numConsecutiveMutations
+        super.init(name: "HybridEngine")
+    }
+
+    override func initialize() {
```

Comment:
Let's add an `assert(fuzzer.config.useAbstractInterpretation, "The HybridEngine requires abstract interpretation to be enabled")` because I think otherwise, this will just completely fail to do anything useful :D